Test/dafny0/TypeConstraints.dfy.expect

```diff
@@ -1,6 +1,6 @@
 TypeConstraints.dfy(42,6): Error: RHS (of type int) not assignable to LHS (of type bool)
 TypeConstraints.dfy(49,6): Error: RHS (of type int) not assignable to LHS (of type bool)
-TypeConstraints.dfy(56,9): Error: RHS (of type int) not assignable to LHS (of type bool)
+TypeConstraints.dfy(56,13): Error: RHS (of type int) not assignable to LHS (of type bool)
```

Comment:
Same reporting problem here. Reporting on ":=" (before) is preferable than on "," but reporting on one of the RHS would have been even better.